tests/unit/Rules/AlwaysInvalidTest.php

```diff
@@ -17,11 +17,20 @@
 
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Rules\AlwaysInvalid
+ *
+ * @author Andreas Wolf <dev@a-w.io>
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author William Espindola <oi@williamespindola.com.br>
  */
-class AlwaysInvalidTest extends TestCase
+final class AlwaysInvalidTest extends TestCase
```

Comment:
`RuleTestCase` :)

Reply:
The same of [AlwaysValid](https://github.com/Respect/Validation/pull/981#discussion_r192490376) but for valid providers